client/pkg/srv/srv_test.go

```diff
@@ -226,9 +226,9 @@ func TestSRVDiscover(t *testing.T) {
 		},
 		{
 			[]*net.SRV{
-				{Target: "a.example.com", Port: 2480},
-				{Target: "b.example.com", Port: 2480},
-				{Target: "c.example.com", Port: 2480},
+				{Target: "a.example.com.", Port: 2480},
```

Comment:
Should we leave a test case that doesn't have `.` suffix to ensure we don't introduce regression in future?